packages/react-search-ui/src/SearchProvider.js

```diff
@@ -37,11 +37,19 @@ class SearchProvider extends Component {
     })
   };
 
+  subscription = state => {
+    this.setState(state);
+  };
```

Comment:
This `;` seems out of place? You don't finish the other functions with a semicolon.

Reply:
@omegahm All of semi-colons are managed by Prettier, so It's actually put there by Prettier! 

And in this case it's a "Public Field Definition" as opposed to a method definition like the other functions, so it gets a semi-colon, while the others don't. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Public_field_declarations